TodoList/bot/services/botService.ts

```diff
@@ -122,7 +122,7 @@ export async function createMyTodosResponse(context: TurnContext, pageNow: numbe
         });
         myTodosResp.tab.value.cards.push({"card": todoListPayload});
     }
-
+    console.log(JSON.stringify(myTodosResp).length);
```

Comment:
We may use a logger to log different level information. We don't need to switch to logger in this PR. But you can see is there any logs are not used any more and can be removed directly.